.github/workflows/test-setup.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: build (cached)
         uses: ./.github/workflows/reusable/cached-build
+      - run: npm run check
+        working-directory: 'packages/${{ inputs.package }}'
```

Comment:
Notice that `check` is done deliberately before spinning up docker services. No point in starting up services if tests don't compile :shrug: